platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -699,6 +699,11 @@ public VirtualRootNode<K, V> copy() {
     protected void destroyNode() {
         if (pipeline != null) {
             pipeline.destroyCopy(this);
+        } else {
+            logger.info(
```

Comment:
Is it possible to somehow identify the corresponding data source? Or maybe this log message is not needed at all.

Reply:
Added node route information to the log message.